Makefile

```diff
@@ -15,58 +13,58 @@ export GITHUB_PAGES_REPO ?= KohlsTechnology/eunomia
 export HELM_CHARTS_SOURCE ?= deploy/helm/eunomia-operator
 export HELM_CHART_DEST ?= $(GITHUB_PAGES_DIR)
 
-LDFLAGS := "-X github.com/KohlsTechnology/eunomia/version.Version=$(VERSION) \
-	-X github.com/KohlsTechnology/eunomia/version.Vcs=$(BUILD_COMMIT) \
+LDFLAGS := "-X github.com/KohlsTechnology/eunomia/version.Vcs=$(BUILD_COMMIT) \
 	-X github.com/KohlsTechnology/eunomia/version.Timestamp=$(BUILD_TIMESTAMP) \
 	-X github.com/KohlsTechnology/eunomia/version.Hostname=$(BUILD_HOSTNAME)"
 
-all: manager
+.PHONY: all build
 
-# Run tests
-native-test: generate fmt vet
-	go test ./pkg/... ./cmd/... -coverprofile cover.out
+all: build
 
-# Build manager binary
-manager: generate fmt vet
-	go build -o build/_output/bin/eunomia  -ldflags $(LDFLAGS) github.com/KohlsTechnology/eunomia/cmd/manager
+clean:
+	rm -rf build/_output
 
-# Build manager binary
-manager-osx: generate fmt vet
-	GOOS=darwin go build -o build/_output/bin/eunomia -ldflags $(LDFLAGS) github.com/KohlsTechnology/eunomia/cmd/manager
+# Build binary
+build:
+	GO111MODULE=on go mod vendor
+	GO111MODULE=on go build -o build/_output/bin/eunomia -ldflags $(LDFLAGS) github.com/KohlsTechnology/eunomia/cmd/manager
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet
+run:
 	go run ./cmd/manager/main.go
 
+test: fmt lint vet test-unit test-e2e
+
+test-e2e:
+	./scripts/e2e-test.sh
+
+test-unit:
+	./scripts/unit-tests.sh
+
 # Install CRDs into a cluster
 install:
 	cat deploy/crds/*crd.yaml | kubectl apply -f-
 
-# Run go fmt against code
+# Run gofmt against code
 fmt:
-	go fmt ./pkg/... ./cmd/...
+	test -z "$(shell gofmt -l . | grep -v ^vendor)"
```

Comment:
I'm confused by this block; when running `make fmt` I would be surprised that it's just a *test* instead of actually reformatting my code; also, the comment above says "Run gofmt", but running gofmt would normally reformat my code. Before this change, there was a "check-gofmt" target which did a `test` call, and which was de facto now renamed to "fmt" — why this change in the target's name?